mabl snippets/getLatestTwilioSms.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * This snippet fetches the last SMS message from Twilio and returns the body.
+ * You will need to place your Twilio API credentials in a mabl Credential, and add it to the Plan before running the snippet.
+ * Add a phone number in 'toNumberFilter' below, to filter to only a specific inbound phone number.
+ *
+ * @param {object} mablInputs - Object containing mabl inputs such as variables (mablInputs.variables).
+ *                              Use mablInputs.variables.user for user defined variables
+ https://messages.google.com/web/conversations/new *                              (For example myVar may be accessed as mablInputs.variables.user.myVar)
+ *
+ * @param {function} callback - A callback function that must be called to complete
+ *                              the javascript step and provide a value to the following
+ *                              steps of the flow/journey. A return statement from this
+ *                              function call will not provide any results for use
+ *                              in the following steps in this flow or journey.
+ */
+function mablJavaScriptStep(mablInputs, callback) {
+
+  // Add Twilio API creds as username: Account SID, password: auth token, add to the plan
+  var twilioAccountSid = mablInputs.variables.web.defaults.credentials.username;
+  var twilioAccountAuthToken = mablInputs.variables.web.defaults.credentials.password;
```

Comment:
Tests can only ever use 1 set of credentials per run. If a test uses our Credentials feature to login then they would not be able to use them here. Might make sense to outline these as environment variables or just name the variable here instead.

Reply:
Configs broken out with alt example.